### ISSUE TITLE
Replace "Cicero" and "Ergo" headers with logos

### ISSUE THREAD

**Issue by Michael-Grover** [accordproject/techdocs#112]
At the top of the docs page, have four boxes calling out different aspects of the tech docs. The first two are Cicero and Ergo. We currently use text to spell out Cicero and Ergo, however we have logos for those two projects.

![image](https://user-images.githubusercontent.com/40569793/66152602-ce213680-e5e7-11e9-8b3d-312ffa61febf.png)

Replace this text with the official Cicero and Ergo logos. Make sure there is alt text so that visually impaired people can still identify what these projects are called.

It should look something like this:
![image](https://user-images.githubusercontent.com/40569793/66153054-c8782080-e5e8-11e9-95c8-6dfe11ecead3.png)

Logo SVGs:
[logo svgs.zip](https://github.com/accordproject/techdocs/files/3687487/logo.svgs.zip)

**Comment by jeromesimeon** [comment]
Not sure how easy this one is. I suspect this is part of the metadata used by docusaurus, not an HTML component.

**Comment by Michael-Grover** [comment]
Thank you @lex111 , closing the issue.

**Comment by lex111** [comment]
I look forward to new issues! They will?

**Comment by Michael-Grover** [comment]
@lex111 yes we should be adding new issues throughout the month.

**Comment by lex111** [comment]
@Michael-Grover Great! :+1:
